hapi/hedera-protobufs/services/consensus_create_topic.proto

```diff
@@ -70,4 +71,37 @@ message ConsensusCreateTopicTransactionBody {
      * If specified, there must be an adminKey and the autoRenewAccount must sign this transaction.
      */
     AccountID autoRenewAccount = 7;
+
+    /**
+     * Access control for update/delete of custom fees.
+     * <p>
+     * If unset, custom fees CANNOT be set for this topic.<br/>
+     * If not set when the topic is created, this field CANNOT be set via update.
+     * If set when the topic is created, this field CAN be changed via update.
+     */
+    Key fee_schedule_key = 8;
+
+    /**
+     * A set of keys that are allowed to submit messages to the topic without
+     * paying the topic's custom fees.
+     * <p>
+     * If a submit transaction is signed by _any_ key included in this set,
+     *  custom fees SHALL NOT be charged for that transaction.
+     * <p>
+     * If fee_exempt_key_list is unset, the following keys are exempt
+     *  from custom fees: adminKey, submitKey, fee_schedule_key.
```

Comment:
Are these keys exempt from custom fees _only_ when the list is empty or unset?
I would think it makes more sense to exempt the admin, submit, and fee schedule keys
from custom fee assessment regardless of the content of this list...

Reply:
Hi guys, we don't want implicit exclusions from the fee.
On the network level, only the keys included in the Fee Exempt Key List are exempt.
To make life easier on the application development side, if the list is not specified, the SDK will take care to include by default the Admin Key, Submit Key and Fee Schedule Key in that list.

Reply:
Please note[ this is the latest version of the HIP](https://github.com/hashgraph/hedera-improvement-proposal/pull/1045), where the list name is now Fee Exempt Key List.